app/main/posts/savedsearches/mode-context.html

```diff
@@ -3,7 +3,7 @@
         <ol class="breadcrumbs" ng-controller="navigation as nav">
             <li><a href="/">{{nav.site.name}}</a></li>
         </ol>
-        <h1 class="mode-context-title">
+        <h1 class="mode-context-title" translate>
```

Comment:
What are we translating here?

Reply:
Ah, the saved-search name. When we create a deployment, we also create 3 saved searches, one for the posts from the users, one for unstructured posts and one for all published posts. This is preparation for those strings, but they will come from the api.

Reply:
I think this will strip the icon when translated. So
1. you need to wrap just the search name
2. the user should really just edit the saved search name. I don't think we should be trying to translate user editable content.